inventory_management_system_api/models/catalogue_item_property_template.py

```diff
@@ -0,0 +1,27 @@
+"""
+Module for defining the database models for representing a Catalogue item property template
+"""
+
+from typing import Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from inventory_management_system_api.models.catalogue_category import AllowedValues
+from inventory_management_system_api.models.custom_object_id_data_types import StringObjectIdField
+
+# pylint: disable=duplicate-code
```

Comment:
Could you enable this at the end of the file again, its easy to forget about if the file is added to.